CHANGELOG.md

```diff
@@ -16,10 +16,12 @@
 - [In Progress: 06 Jul 2021] Add support for finding a patient online from ID scan
 
 ### Changes
-- [In Progress: 17 Jun 2021] Overdue list improvements
+- [In Progress: 7 Jul 2021] Overdue list improvements
   - Change overdue list UI
   - Load overdue patients without phone number
   - Add option to download & share overdue list
+  - Update `ContactPatientBottomSheet` Behaviour
+    - Updated contact patient bottom sheet UI
```

Comment:
Can you update the CHANGELOG to just mention the changes that will be visible to users here, please? For example, I don't see the point of having a sub-point for the behaviour change, we can just say `Update contact patient bottom sheet UI`. The Features, Changes & Fixes section are something that is visible to users, so we need to make sure we are writing the messages that are indicative of those changes here and useful for users.